### PR TITLE
Support installing in org namespace with child apps in cluster namespace

### PR DESCRIPTION
**Pull request by tfussell**
This PR:

- Allows the default-apps-openstack app to be installed in the organization namespace while the child apps are installed in the cluster namespace. This is currently required because the WC kubeconfig is in the organization namespace while the cluster-values are in the cluster namespace.

Example `values.yaml`:
```
cluster:
  name: thomas40
  namespace: thomas40

organization:
  name: giantswarm-2
  namespace: org-giantswarm-2

appOperator:
  version: 5.2.0
```

### Testing

- [x] Fresh install works.

Not adding a changelog as there haven't been any releases yet.

### CODE REVIEW

File: helm/default-apps-openstack/templates/calico.yaml [hunk 1]
```diff
@@ -6,11 +6,11 @@ metadata:
...
     {{- include "labels.common" . | nindent 4 }}
   name: {{ .Values.cluster.name }}-calico
-  namespace: {{ .Release.Namespace }}
+  namespace: {{ .Values.cluster.namespace }}
 spec:
   catalog: default
   {{- include "kubeconfig" . | nindent 2 }}
   {{- include "config" . | nindent 2 }}
-  name: calico-app
+  name: calico
```
Comment (tfussell) [inline helm/default-apps-openstack/templates/calico.yaml:14]: Unrelated bug fix. Unsure why the suffix isn't necessary, but this is the name of the tarball in the app catalog.

**Comment by tfussell** [comment]
Tested on `gravity` with `helm install thomas40-default-apps helm/default-apps-openstack -n org-giantswarm-2 --values values.yaml`

**Review by yulianedyalkova — APPROVED** [review]

File: helm/default-apps-openstack/values.yaml [hunk 2]
```diff
@@ -1,6 +1,10 @@
 cluster:
   name: ""
-  organization: ""
+  namespace: ""
```
Comment (MarcelMue) [inline helm/default-apps-openstack/values.yaml:3]: We should ideally have all in the same namespace - so I would rather work on making that work TBH

File: helm/default-apps-openstack/templates/calico.yaml (hunk 1 above)
Comment (rossf7) [reply helm/default-apps-openstack/templates/calico.yaml:14]: The naming should match the tarball as app-operator uses this to build the tarball URL in the chart CR.

I opened #4 for this. Thanks for the heads up @yulianedyalkova :)

File: helm/default-apps-openstack/values.yaml (hunk 2 above)
Comment (rossf7) [reply helm/default-apps-openstack/values.yaml:3]: Yes testing with https://github.com/giantswarm/cluster-apps-operator/pull/148 on gamma looks promising.

I agree with @MarcelMue  better to work in this direction.

Comment (kopiczko) [reply helm/default-apps-openstack/values.yaml:3]: How about removing `org-` prefix from the `organization.namespace` instead?

Comment (tfussell) [reply helm/default-apps-openstack/values.yaml:3]: Unfortunately the namespace name and the organization name (plus `org-` prefix) aren't always the same due to some organization names containing uppercase letters and possibly other characters that aren't allowed in namespace names.

Comment (tfussell) [reply helm/default-apps-openstack/values.yaml:3]: I did it this way temporarily so we could test with our existing operators. Fine if we want to wait and do it the right way though.

**Comment by tfussell** [comment]
Waiting for https://github.com/giantswarm/app-operator/pull/902

**Comment by tfussell** [comment]
No longer needed after moving apps to org namespace.
